proto/beacon/p2p/v1/messages.pb.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Revert this file? I don't see any changes in messages.proto

Reply:
Why did git make it show up? Is it because i regenerated it?

Reply:
You committed it, yes. 

There is nothing interesting changed here and this change is not necessary

Reply:
What's the recommended way to do this?